lib/cocoapods-core/cdn_source.rb

```diff
@@ -16,8 +18,8 @@ def initialize(repo)
       @startup_time = Time.new
 
       @executor = Concurrent::ThreadPoolExecutor.new(
-        :min_threads => 1,
-        :max_threads => 200,
+        :min_threads => 5,
+        :max_threads => (ENV['MAX_CDN_NETWORK_THREADS'] || 50).to_i,
```

Comment:
could `50` be put into a constant similar to `MAX_NUMBER_OF_RETRIES`?

Reply:
Done.